src/com/sap/piper/integration/TransportManagementService.groovy

```diff
@@ -0,0 +1,136 @@
+
+        }
+
+        if (config.verbose) {
+
+            script.echo response
+
+        }
+
+        def oAuthToken = jsonUtils.jsonStringToGroovyObject(response).access_token
+
+        script.echo "[TransportManagementService] OAuth Token retrieved successfully."
+
+        return oAuthToken
+
+    }
+
+
+    def uploadFileToTMS(String url, String token, String file, String namedUser) {
+
+        script.echo "[TransportManagementService] File upload started."
+
+        if (config.verbose) {
+
+            script.echo "[TransportManagementService] URL: '${url}', File: '${file}'"
+
+        }
+
+        def httpResponse = script.sh returnStdout: true,
+            script: """#!/bin/sh -e
+                        curl -XPOST -H 'Authorization: Bearer ${token}' -F 'file=@${file}' -F 'namedUser=${namedUser}' -o responseFileUpload.txt --write-out '%{http_code}' '${url}/v2/files/upload'
+                    """
+
+        def response = script.readFile("responseFileUpload.txt")
+
+        if (httpResponse.toInteger() < 200 || httpResponse.toInteger() >= 300) {
+
+            script.error "[TransportManagementService] File upload failed. HTTP-Status: ${httpResponse} \n [ERROR] Response: '${response}'"
+
+        }
+
+        if (config.verbose) {
+
+            script.echo response
+
+        }
+
+        def fileUploadDetails = jsonUtils.jsonStringToGroovyObject(response)
+
+        script.echo "[TransportManagementService] File upload successful."
+
+        return fileUploadDetails
+
+    }
+
+
+    def uploadFileToNode(String url, String token, String nodeName, int fileId, String description, String namedUser) {
+
+        script.echo "[TransportManagementService] Node upload started."
+
+        if (config.verbose) {
+
+            script.echo "[TransportManagementService] URL: '${url}', NodeName: '${nodeName}', FileId: '${fileId}'"
+
+        }
+
+        def bodyMap = [nodeName: nodeName, contentType: 'MTA', description: description, storageType: 'FILE', namedUser: namedUser, entries: [[uri: fileId]]]
+
+        def body = jsonUtils.groovyObjectToPrettyJsonString(bodyMap)
+
+        def httpResponse = script.sh returnStdout: true,
+            script: """#!/bin/sh -e
+                        curl -XPOST -H 'Authorization: Bearer ${token}' -H 'Content-Type: application/json' -d '${body}' -o responseNodeUpload.txt --write-out '%{http_code}' '${url}/v2/nodes/upload'
+                    """
+
+        def response = script.readFile("responseNodeUpload.txt")
+
+        if (httpResponse.toInteger() < 200 || httpResponse.toInteger() >= 300) {
+
+            script.error "[TransportManagementService] Node upload failed. HTTP-Status: ${httpResponse} \n [ERROR] Response: ${response}"
+
+        }
+
+        if (config.verbose) {
+
+            script.echo response
+
+        }
+
+        def nodeUploadDetails = jsonUtils.jsonStringToGroovyObject(response)
+
+        script.echo "[TransportManagementService] Node upload successful."
```

Comment:
You could extract `script.echo/error "[TransportManagementService] $msg"` to a `echo` and `error` method.

Reply:
Done.